src/Tizen.NUI/src/public/BaseComponents/View.cs

```diff
@@ -224,6 +226,16 @@ internal View(global::System.IntPtr cPtr, bool cMemoryOwn, bool shown = true) :
 
             GrabTouchAfterLeave = defaultGrabTouchAfterLeave;
             AllowOnlyOwnTouch = defaultAllowOnlyOwnTouch;
+
+            var hoverEnv = System.Environment.GetEnvironmentVariable("DISABLE_HOVERING");
```

Comment:
IMHO, I don't think it have to get environment variable and parse the value every time a View is created. How about making the `disableHoverEvent` to a static variable and calcuate once when it assigned in the first time?